python/lsst/ts/ess/common/device/base_device.py

```diff
@@ -94,6 +94,19 @@ async def __aexit__(
     ) -> None:
         await self.close()
 
+    def __repr__(self) -> str:
+        return self.__str__()
```

Comment:
Minor point, but: I am pretty sure if you define `__repr__` and leave `__str__` undefined then the default `__str__` uses repr.

Reply:
You are right. I removed the `__str__` methods and only implemented `__repr__` now.